src/input.cpp

```diff
@@ -10,6 +10,7 @@
 #include "translations.h"
 #include "catacharset.h"
 #include "options.h"
+#include "ui.h"
```

Comment:
Strange include, why do you need ui.h here? For `WINDOW*` would only need cursesdef.h.